assets/js/blocks/cart-checkout/payment-methods/express-payment-methods.js

```diff
@@ -48,32 +55,71 @@ const ExpressPaymentMethods = () => {
 			setPaymentStatus,
 		]
 	);
+
+	/**
+	 * onExpressPaymentClose should be triggered when the express payment process is cancelled or closed.
+	 *
+	 * This restores the active method and returns the state to pristine.
+	 */
 	const onExpressPaymentClose = useCallback( () => {
```

Comment:
When we close express payment, we restore previous payment method and set pristine.